app/src/main/java/org/oppia/app/viewmodel/ViewModelBridgeFactory.kt

```diff
@@ -12,13 +12,15 @@ import javax.inject.Provider
  */
 class ViewModelBridgeFactory<V : ViewModel> @Inject constructor(
   private val viewModelProvider: Provider<V>
-): ViewModelProvider.Factory {
+) : ViewModelProvider.Factory {
   override fun <T : ViewModel?> create(modelClass: Class<T>): T {
     val viewModel = viewModelProvider.get()
     // Check whether the user accidentally switched the types during provider retrieval. ViewModelProvider is meant to
     // guard against this from happening by ensuring the two types remain the same.
     check(modelClass.isAssignableFrom(viewModel.javaClass)) {
+      /* ktlint-disable max-line-length */
```

Comment:
Can you check if a single line comment rule works? i.e `// ktlint-disable max-line-length` next to the line

Reply:
Ok, it's working, I tried once it doesn't work and there are some issues in ktlint which says it doesn't work for them so I thought it would be good to go with this blocky thing. 
Updated with one line.